tests/src/utils/test_rlp.py

```diff
@@ -26,11 +26,25 @@ def test_should_match_prefix_reference_implementation(self, cairo_run, data):
 
             assert output == [expected_type, expected_offset, expected_len]
 
+        def test_should_raise_when_data_len_is_zero(self, cairo_run):
+            with cairo_error("RLP data is empty"):
+                cairo_run("test__decode_type", data=[])
+
+        @pytest.mark.parametrize("prefix", [0xB8, 0xF8])
```

Comment:
Prefer hypothesis 
Should there be a test also for when the prefix is > ff ?

Reply:
It falls under the prefix >0xf7 so it should be fine